docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| sink.buffer-flush.max-rows        | No           | 500000            | The maximum number of rows that can be accumulated in memory before being sent to StarRocks at a time. Only available for version `V1` when using at-least-once. Valid values: 64000 to 5000000.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| sink.buffer-flush.interval-ms     | No           | 300000            | The interval at which data is flushed. Only available for at-least-once mode. Valid values: 1000 to 3600000. Unit: ms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| sink.max-retries                  | No           | 3                 | The number of times that the system retries to perform the Stream Load. Only available for sink version `V1` currently. Valid values: 0 to 10.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/7f91cf54-0eec-4aef-bdbc-f3389603989d)
I guess it should be "the Stream Load job"?